config/disabled.yml

```diff
@@ -44,6 +44,12 @@ Style/MissingElse:
     - case
     - both
 
+Style/Send:
+  Description: 'Prefer `BasicObject#__send__` or `Object#public_send` to `send`,
+                as `send` may overlap with existing methods.'
+  StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#prefer-public-send'
+  Enabled: false
+
```

Comment:
Updated `description` and linked the relevant `style guide`. The rule about preferring `__send__` over `send` will appear beneath `prefer-public-send`, so it's intentional that I link to `prefer-public-send`.